tests/TestCase/Routing/RouterTest.php

```diff
@@ -2491,30 +2491,6 @@ public function testCustomRouteException()
         Router::connect('/:controller', [], ['routeClass' => 'Object']);
     }
 
-    /**
-     * test reversing parameter arrays back into strings.
-     *
-     * Mark the router as initialized so it doesn't auto-load routes
-     *
-     * @return void
-     */
-    public function testReverseToken()
```

Comment:
Shouldn't need method retained and just the extra key in $params array removed? Or there's already another test for the same?

Reply:
#13669 removed `'_Token' => ['key' => 'sekret'],` and based on test name, it was object of test, so I removed completely.